.github/workflows/pytest.yml

```diff
@@ -64,3 +64,4 @@ jobs:
           SERVER_HOST: http://localhost
           POSTGRES_TEST_DB: test_direct4ag
           POSTGRES_DB: direct4ag
+          GEOSTREAMS_API_STR: https://direct4ag.ncsa.illinois.edu/geostreams/api/
```

Comment:
Does it need the actual value or would a dummy value work here? We could make this a organization secret so we aren't putting the URL in the repo

Reply:
I wanted to keep a dummy here but I don't have enough permissions so could you please add a secret and update the file?

Reply:
we can do the {{}} thing here...

Reply:
I'm confused, if I put GEOSTREAMS_API_STR: http://localhost/geostreams/api will the tests work? If so, why not just insert a dummy value. If it needs a real value, then we should use a secret.